packages/ez_pgocaml/ez_pgocaml.1.0.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "A simple library to work with pgocaml"
+maintainer: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+authors: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+license: "GPL-3.0-or-later"
```

Comment:
This says GPLv3, but the repo says LGPLv2.1+linking exception. https://github.com/OCamlPro/ez_pgocaml/blob/master/LICENSE.md